src/components/Main/MainView/MainViewSidebar/ContentEditor.vue

```diff
@@ -80,6 +80,8 @@ export default defineComponent({
 .content {
   width: 100%;
   white-space: pre-line;
+  overflow: auto;
+  overflow-wrap: break-word;
```

Comment:
```suggestion
  word-break: keep-all;
  overflow-wrap: break-word;
  min-width: 0;
```
できれば単語切りたくないのでこっちでお願いします:pray:

https://w3g.jp/blog/confusing_word-break_word-wrap
`min-width: 0`はこのあたりです(flexもgridも同じような感じです)
https://qiita.com/mpyw/items/dfc63c1fed5dfc5eda26

Reply:
ありがとうございます :pray:
修正しました